noxfile.py

```diff
@@ -22,6 +22,14 @@ def omegaconf(session):
     session.run("pytest")
 
 
+@nox.session(python="pypy")
+def test_omegaconf_pypy(session):
+    deps(
+        session, editable_installl=False, requirements="requirements/dev_pypy.txt"
+    )  # ensure we test the regular install
+    session.run("pytest", "-Wdefault")
```

Comment:
The `-Wdefault` flag here allows warnings in the pypy session (whereas in other sessions, warnings would cause pytest to fail).
In the pypy session, a warning is being caused by the `import yaml` statement:
`ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__`